esm/api/v4/product.js

```diff
@@ -19,6 +21,7 @@ export function getAllZones() {
  * GET /products/zones
  * @param {Object} params
  * @param {String} params.undefined
```

Comment:
nitpick: there's an undefined param here, also params are not needed for this function